test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -24,26 +24,65 @@ test('IndexTable - bulk action', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
   await indexTablePage.goto();
 
+  let isCheckboxCorrect: boolean, tempText: string | null;
```

Comment:
can we rename this to `isCheckboxValuesCorrect` and `selectAllTableHeadText` so it's easier to know what they are from the variable names.